### ISSUE TITLE
How to update my local install of pico-sdk?

### ISSUE THREAD

**Issue by seamusdemora** [raspberrypi/pico-sdk#1119]
I probably cannot see the forest for the trees, but...

After "installing" the pico-sdk on my Ubuntu host by (generally) following the instructions, how do I keep it up-to-date? Have I missed something in the documentation - or the README for this repo?

**Comment by lurch** [comment]
See section 2.3 in https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf :slightly_smiling_face:

**Comment by seamusdemora** [comment]
Oops! Thank you - I missed that  :P   Does that update the Examples also?

**Comment by lurch** [comment]
No, you'd also need to do a `git pull` in the pico-examples directory.
